File: latihan b/Latihan2 Py/Memori.py
from Hardware import Hardware

#kelas yang digunakan untuk mengimplementasikan kelas Memory
class Memori(Hardware):

	#konstruktor
	def __init__(self):
		self.frekuensi = 0
		self.Memorisize = 0
		self.support = ""

	#getter dan setter untuk atribut frekuensi
	def setfrekuensi(self, frekuensi):
		self.frekuensi = frekuensi

	def getfrekuensi(self):
		return self.frekuensi

	#getter dan setter untuk atribut memorySize
	def setMemorisize(self, Memorisize):
		self.Memorisize = Memorisize

	def getMemorisize(self):
		return self.Memorisize

	#getter dan setter untuk atribut supportsCuda
	def setsuport(self, support):
		self.support = support

	def getsupport(self):
		return self.support

File: latihan b/latihan2 PHP/Memori.php
<?php
// import file kelas parent
include "Hardware.php";

class Memori extends Hardware
{ 
/*kelas yang digunakan untuk mengimplementasikan memori/
/*atribut Memori*/
		private $frekuensi = 0;		//frekuensi
		private $Memorisize = 0;	//Memorisize
		private $support = "";		//support
		
		 // constructor dengan parameter
        public function __construct($frekuensi = 0, $Memorisize = 0, $support=""){
            $this->frekuensi = $frekuensi;
            $this->Memorisize = $Memorisize;
            $this->support = $support;
        }
	
	/*mengeset nilai frekuensi*/
	public function setfrekuensi($frekuensi)
	{
		$this->frekuensi = $frekuensi;
	}
	/*mengeset nilai Memorisize*/
	public function setMemorisize($Memorisize)
	{
		$this->Memorisize = $Memorisize;
	}
	
	/*mengeset nilai support*/
	public function setsuport($support)
	{
		$this->support = $support;
	}

	/*mengembalikan nilai frekuensi*/
	public function getfrekuensi()
	{
		return $this->frekuensi;
	}
	/*mengembalikan nilai memorisize*/
	public function getMemorisize()
	{
		return $this->Memorisize;
	}
	
	/*mengembalikan nilai support*/
	public function getsupport()
	{
		return $this->support;
	}
	
	function __destruct(){
		/*destuktor*/
	}
}

?>